README.md

```diff
@@ -187,9 +187,10 @@ For non-interactive usage one can use the batch install option:
 start /wait "" Miniforge3-Windows-x86_64.exe /InstallationType=JustMe /RegisterPython=0 /S /D=%UserProfile%\Miniforge3
 ```
 
-### Unix-like platforms (macOS & Linux)
+### Unix-like platforms (macOS, Linux, WSL)
+From a terminal window, download the installer appropriate for your computer's architecture using curl or wget or your favorite program. (WSL users should download the appropriate Linux installer.)
```

Comment:
Can you spell out what WSL means

Reply:
I may have gone too far but spelling out WSL in a parenthetical felt wrong.